.travis.yml

```diff
@@ -58,22 +57,23 @@ jobs:
           name:  Build
           script:
             - ./scripts/travis/build/build.sh
-          after_success: ./scripts/ci/utils/artifact-to-s3.sh -a ./dist/demo-shell -o "$S3_DBP_FOLDER/alfresco-demoshell.tar.bz2"
+          after_success:
+            - ./scripts/ci/utils/artifact-to-s3.sh -a ./dist/demo-shell -o "$S3_DBP_FOLDER/alfresco-demoshell.tar.bz2"
+            - ./scripts/ci/utils/artifact-to-s3.sh -a ./lib/dist -o "$S3_DBP_FOLDER/alfresco-libs.tar.bz2"
         - stage: Lint & Build Dist
           name: Lint
           script:
             - ./scripts/lint.sh
 
 
         # Run Only on develop and master
-        - stage: Lint & Build Dist & Release
-          name:  Lint & Build Dist & Release
+        - stage:
+
+          name: Npm Release
+          before_script: ./scripts/ci/utils/artifact-from-s3.sh -a "$S3_DBP_FOLDER/alfresco-libs.tar.bz2" -o "./lib/dist"
```

Comment:
@eromano can you check this change. 
We use the same Lint & Build Dist to build and link the artifact will store in S3 and the Release job will fetch the artifact from S3 and publish it on npm